tests/test_liquid_handler.py

```diff
@@ -188,7 +188,7 @@ def test_distribute_multiple_volumes_multi_channel(self):
             overhead_liquid=False
         )
         # Assert
-        self.lh.p300_multi.aspirate.assert_called_with(volume=100, location=self.source_well)
+        self.lh.p300_multi.aspirate.assert_called_with(volume=100, location=self.source_well, single_tip_mode=False)
```

Comment:
I don't think `single_tip_mode` is an argument of the current `aspirate` implementation, rather, it's an argument for `aspirate_tracked`